src/widgets/youtube/player.tsx

```
@@ -3,94 +3,123 @@
+      stopTracking();
+      playerRef.current?.removeEventListener("onStateChange", _onStateChange);
+    };
   }, []);
 
   return (
     <div ref={containerRef} onMouseDown={loadIFrameElement}>
       <YouTubeEmbed
-        params={enableJsApi ? IframeParams + "&enablejsapi=1" : IframeParams}
-        videoid={video.videoId}
-        playlabel={video.title}
+        params={enableJsApi ? iframeParams + "&enablejsapi=1" : iframeParams}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)